test/javascripts/acceptance/group-manage-interaction-test.js

```diff
@@ -8,6 +10,20 @@ acceptance("Managing Group Interaction Settings", {
 });
 
 QUnit.test("As an admin", async assert => {
+  updateCurrentUser({
+    moderator: false,
+    admin: true,
+    can_create_group: true
+  });
+
+  let groupResponse = _.clone(groupFixtures["/groups/discourse.json"]);
+  groupResponse.group.can_admin_group = true;
+  pretender.get("/groups/discourse.json", () => [
```

Comment:
It might be DRYer to add another fixture to `group-fixtures.js` for a new group that has the `can_admin_group` set to true, rather than cloning and updating it over and over in a handful of tests.

Reply:
Fair enough. I had a heck of a time getting those to work - I should have cut my losses much earlier and just created some new fixtures, as you suggest.